packages/shared/src/graphql/squads.ts

```diff
@@ -65,6 +65,16 @@ type EditSquadOutput = {
   editSquad: Squad;
 };
 
+type SquadMembershipsOutput = {
+  mySourceMemberships: {
+    sources: {
+      node: {
+        source: Squad;
+      };
+    }[];
```

Comment:
I think we have an existing type for this.
```suggestion
    sources: Edge<SquadMember>[];
```

Reply:
Think this also would be:
```
    sources: Edge<Squad>[];
```